test/e2e-tests.sh

```diff
@@ -66,6 +66,19 @@ go_test_e2e -timeout=10m \
   ./test/e2e/autotls || failed=1
 kubectl delete -f ./test/config/autotls/certmanager/selfsigned/
 
+# Certificate conformance tests must be run separately
+kubectl apply -f ./test/config/autotls/certmanager/selfsigned/ -f ./test/config/autotls/certmanager/config-network.yaml
+add_trap "kubectl delete -f ./test/config/autotls/certmanager/selfsigned/ -f ./test/config/autotls/certmanager/config-network.yaml --ignore-not-found" SIGKILL SIGTERM SIGQUIT
+go_test_e2e -timeout=10m \
+  ./test/conformance/certificate/nonhttp01 || failed=1
+kubectl delete -f ./test/config/autotls/certmanager/selfsigned/ -f ./test/config/autotls/certmanager/config-network.yaml
+
+kubectl apply -f ./test/config/autotls/certmanager/http01/ -f ./test/config/autotls/certmanager/config-network.yaml
+add_trap "kubectl delete -f ./test/config/autotls/certmanager/http01/ -f ./test/config/autotls/certmanager/config-network.yaml --ignore-not-found" SIGKILL SIGTERM SIGQUIT
+go_test_e2e -timeout=10m \
+  ./test/conformance/certificate/http01 || failed=1
+kubectl delete -f ./test/config/autotls/certmanager/http01/ -f ./test/config/autotls/certmanager/config-network.yaml
```

Comment:
We probably don't want to delete `config-network` ConfigMap as it could be consumed by the following E2E tests.

Can we directly pass `"cert-manager.certificate.networking.internal.knative.dev"` into E2E test as an environment variable (e.g. `CERTIFICATE_CLASS`) ?

Reply:
Yeah, I think that's the better way to do it. I've added a new flag for that.